### ISSUE TITLE
Lua end to end logging PBFT test failure

### ISSUE THREAD

**Issue by achamayou** [microsoft/CCF#1128]
https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=7906&view=ms.vss-test-web.build-test-results-tab&runId=41746&resultId=100042&paneView=attachments

```
Traceback (most recent call last):
  File "/mnt/build/1/s/tests/e2e_logging.py", line 317, in <module>
    run(args)
  File "/mnt/build/1/s/tests/e2e_logging.py", line 287, in run
    verify=args.package is not "libjs_generic",
  File "/mnt/build/1/s/tests/suite/test_requirements.py", line 19, in wrapper
    return func(*args, **kwargs)
  File "/mnt/build/1/s/tests/suite/test_requirements.py", line 42, in wrapper
    return func(network, args, *nargs, **kwargs)
  File "/mnt/build/1/s/tests/suite/test_requirements.py", line 42, in wrapper
    return func(network, args, *nargs, **kwargs)
  File "/mnt/build/1/s/tests/e2e_logging.py", line 29, in test
    txs.verify(network)
  File "/mnt/build/1/s/tests/infra/logging_app.py", line 55, in verify
    result={"msg": self.pub[pub_tx_index]},
  File "/mnt/build/1/s/tests/infra/checker.py", line 74, in __call__
    self.node_client, rpc_result.commit, rpc_result.term
  File "/mnt/build/1/s/tests/infra/checker.py", line 43, in wait_for_global_commit
    raise TimeoutError("Timed out waiting for commit")
TimeoutError: Timed out waiting for commit
```

**Comment by achamayou** [comment]
Another instance, at the same term (2), view (197):

https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=7906&view=ms.vss-test-web.build-test-results-tab&runId=41748&resultId=100042&paneView=attachments

**Comment by achamayou** [comment]
https://dev.azure.com/MSRC-CCF/CCF/_build/results?buildId=7921&view=ms.vss-test-web.build-test-results-tab

**Comment by achamayou** [comment]
Fixed by #1136
